test/unit/specs/components/DecisioningEngine/decisioningContext.browser.spec.js

```diff
@@ -0,0 +1,166 @@
+});
+
+const proposition = {
+  id: "2e4c7b28-b3e7-4d5b-ae6a-9ab0b44af87e",
+  items: [
+    {
+      schema: "https://ns.adobe.com/personalization/mock-action",
+      data: {
+        hello: "kitty"
+      },
+      id: "79129ecf-6430-4fbd-955a-b4f1dfdaa6fe"
+    }
+  ],
+  scope: "web://mywebsite.com"
+};
+
+const payloadWithCondition = condition => {
+  return {
+    id: "2e4c7b28-b3e7-4d5b-ae6a-9ab0b44af87e",
+    items: [
+      {
+        id: "79129ecf-6430-4fbd-955a-b4f1dfdaa6fe",
+        schema: "https://ns.adobe.com/personalization/json-ruleset-item",
+        data: {
+          content: JSON.stringify({
+            version: 1,
+            rules: [
+              {
+                condition: {
+                  definition: {
+                    conditions: [condition],
+                    logic: "and"
+                  },
+                  type: "group"
+                },
+                consequences: [
+                  {
+                    type: "item",
+                    detail: {
+                      schema:
+                        "https://ns.adobe.com/personalization/mock-action",
+                      data: {
+                        hello: "kitty"
+                      },
+                      id: "79129ecf-6430-4fbd-955a-b4f1dfdaa6fe"
+                    },
+                    id: "79129ecf-6430-4fbd-955a-b4f1dfdaa6fe"
+                  }
+                ]
+              }
+            ]
+          })
+        }
+      }
+    ],
+    scope: "web://mywebsite.com"
+  };
+};
+
+const mockRulesetResponseWithCondition = condition => {
+  return {
+    getPayloadsByType: () => [
+      payloadWithCondition({
+        definition: {
+          conditions: [condition],
+          logic: "and"
+        },
+        type: "group"
+      })
+    ]
+  };
+};
+
+describe("DecisioningEngine:globalContext:browser", () => {
+  let storage;
+  let eventRegistry;
+  let decisionProvider;
+  let onResponseHandler;
+  let applyResponse;
+
+  const mockEvent = { getContent: () => ({}), getViewName: () => undefined };
+
+  const setupResponseHandler = (window, condition) => {
+    const contextProvider = createContextProvider({
+      eventRegistry,
+      window
+    });
+
+    onResponseHandler = createOnResponseHandler({
+      decisionProvider,
+      applyResponse,
+      event: mockEvent,
+      decisionContext: contextProvider.getContext()
+    });
+
+    onResponseHandler({
+      response: mockRulesetResponseWithCondition(condition)
+    });
+  };
+
+  beforeEach(() => {
+    storage = jasmine.createSpyObj("storage", ["getItem", "setItem", "clear"]);
+    eventRegistry = createEventRegistry({ storage });
+    decisionProvider = createDecisionProvider();
+    applyResponse = jasmine.createSpy();
+  });
+
+  it("satisfies rule based on matched browser", () => {
+    setupResponseHandler(mockWindow({}), {
+      definition: {
+        key: "browser.name",
+        matcher: "eq",
+        values: ["chrome", "safari"]
```

Comment:
test are better with more specificity.  please narrow the matcher to just values: ["chrome"].  It makes it easier to read and see what is being tested.